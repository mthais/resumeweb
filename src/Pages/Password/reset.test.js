import React from 'react'
import { mount } from 'enzyme'
import ResetPasswordPage from './reset'

describe('ResetPasswordPage', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(<ResetPasswordPage />)
  })

  it('should pass with snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
