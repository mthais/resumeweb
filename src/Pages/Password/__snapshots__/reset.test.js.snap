// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordPage should pass with snapshot 1`] = `"<div class=\\"col\\"><h2 class=\\"text-center page-title\\">Redefinir Senha</h2><p class=\\"text-wrapper\\">Digite abaixo sua nova senha!</p><form><div class=\\"position-relative form-group\\"><label for=\\"password\\" class=\\"mb-0\\">Nova Senha</label><div><input name=\\"password\\" id=\\"password\\" type=\\"password\\" class=\\"form-control form-control\\" value=\\"\\"><div class=\\"invalid-feedback\\"></div></div></div><div class=\\"position-relative form-group\\"><label for=\\"password\\" class=\\"mb-0\\">Confirmar Senha</label><div><input name=\\"confirmPassword\\" id=\\"confirmPassword\\" type=\\"password\\" class=\\"form-control form-control\\" value=\\"\\"><div class=\\"invalid-feedback\\"></div></div></div><button disabled=\\"\\" class=\\"px-4 float-right btn btn-primary disabled\\">Redefinir Senha</button></form></div>"`;
